docs/root/configuration/access_log.rst

```diff
@@ -312,3 +312,28 @@ The following command operators are supported:
   TCP
     String value set on ssl connection socket for Server Name Indication (SNI)
 
+%DOWNSTREAM_LOCAL_URI_SAN%
```

Comment:
Not directly actionable in this PR perhaps, but need to think about how to structurally express those field in logging format and keep file access log feature parity with gRPC. We'll likely have upstream TLS set for this as well, and other properties which is defined in `accesslog.proto`.

Some random idea would be making the proto representation as first class, and have some expression in file access log to represent extractor from there.